src/sentry/seer/similarity/utils.py

```diff
@@ -1,14 +1,15 @@
 import logging
 from typing import Any
 
+from sentry import options
 from sentry.constants import PLACEHOLDER_EVENT_TITLES
 from sentry.eventstore.models import Event
 from sentry.utils.safe import get_path
 
 logger = logging.getLogger(__name__)
 
 MAX_FRAME_COUNT = 50
-SEER_ELIGIBLE_PLATFORMS = frozenset(["python", "javascript", "node"])
+SEER_ELIGIBLE_PLATFORMS = options.get("seer.similarity.supported_platforms")
```

Comment:
this doesn't work for a couple reasons:

- this attempts to access the database at import time when it is not available
- the value will not change as options are changed because it is only set at import time